source/portable/NetworkInterface/STM32Fxx/NetworkInterface.c

```diff
@@ -1308,6 +1308,11 @@ static void prvEMACHandlerTask( void * pvParameters )
         {
             /* Something has changed to a Link Status, need re-check. */
             prvEthernetUpdateConfig( pdFALSE );
+
+            if( xGetPhyLinkStatus() == pdFALSE )
+            {
+                FreeRTOS_NetworkDown();
+            }
```

Comment:
If you could change it to:
~~~
#if( ipconfigSUPPORT_NETWORK_DOWN_EVENT != 0 )
{
    if( xGetPhyLinkStatus() == pdFALSE )
    {
        FreeRTOS_NetworkDown();
    }
}
#endif /* ( ipconfigSUPPORT_NETWORK_DOWN_EVENT != 0 ) */
~~~

And define the new macro in `FreeRTOSIPConfigDefaults.h` as 0, along with some comment of what it stands for?

Thank you!